### DISCUSSION TITLE
Is a RPi 3B sufficient to run Nixie

### DISCUSSION THREAD

**Discussion by Msweetmo** [tagyoureit/nodejs-poolController#1021] [Nixie]
Or is a RPi 4 the minimum requirement as per the instructions? If anyone has been running on a RPi 3B or below and has had any issues please let me know. Thanks for the input.

**Comment by tagyoureit** [comment]
3b is perfectly fine for running the software.  The default startup is to re-build the app each time, which takes the longest, but you can skip that part and rebuild manually if you pull, to save time.

**Comment by dkossman** [comment]
3b will be perfectly fine.  I was actually running on a Pi Zero W for a long time and it worked OK.
